db/fixtures/miq_event_definitions.csv

```diff
@@ -209,3 +209,10 @@ containerreplicator_successfulcreate,Replicator Successfully Created Pod,Default
 containerreplicator_compliance_check,Replicator Compliance Check,Default,compliance
 containerreplicator_compliance_passed,Replicator Compliance Passed,Default,compliance
 containerreplicator_compliance_failed,Replicator Compliance Failed,Default,compliance
+
+#
+# Physical Servers Operations
+#
+phs_shutdown,Physical Server Shutdown,Default,phs_operations
+phs_start,Physical Server Start,Default,phs_operations
+phs_reset,Physical Server Reset,Default,phs_operations
```

Comment:
Looking through this file again I have to ask why we are abbreviating these at all?  The existing pattern in this file would lead me to say the event names should be: `physical_server_*`